### ISSUE TITLE
Some confusion about the paper and code

### ISSUE THREAD

**Issue by wuhu-make** [sail-sg/edp#2]
Hi, I am new to the field of reinforcement learning. You guys mentioned in the Efficiency section of the paper that each iteration will only go through the network once. I have the code for the GPT to interpret and the GPT tells me that it goes through the Q-network and the policy network a couple times per iteration, so I'm wondering where the code is that only goes through the network once. There is also the mention that Diffusion-QL requires forward and backward neural networks K times. Is this K a time step or what does it mean

**Comment by Jinbo-He** [comment]
In the training process, the noise is computed only once. First, a sample (s, a, s’) is drawn from the buffer. Then, a is corrupted into $a_t$ by Equation 8, and the diffusion loss $\epsilon - \epsilon_\theta(a_t, t; s)$ is calculated. Next, a is approximated from the predicted noise using Equation 9. Finally, Q(s, a) is computed for policy iteration. This is just my personal understanding, and I hope it can be helpful! ^_^
![image](https://github.com/user-attachments/assets/acc04c39-c4a1-4357-9aaf-a28e96a10714)
